app/models/concerns/audit_extensions.rb

```diff
@@ -73,6 +73,7 @@ def self.auditable_type_complete_values
 
     before_save :fix_auditable_type, :ensure_username, :ensure_auditable_and_associated_name
     before_save :filter_encrypted, :if => Proc.new {|audit| audit.audited_changes.present?}
+    before_save :filter_passwords, :if => Proc.new {|audit| audit.audited_changes.try(:has_key?,'password')}
```

Comment:
missing space before `'password'`, for some time `has_key?` was considered deprecated but from what I've read recently, rubocop ignores it again as matz confirmed, it will be present in ruby 3